executorlib/interactive/shared.py

```
@@ -179,14 +179,16 @@ class InteractiveStepExecutor(ExecutorBase):
 
     def __init__(
         self,
-        max_cores: int = 1,
+        max_cores: Optional[int] = None,
+        max_workers: Optional[int] = None,
         executor_kwargs: dict = {},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)